src/lib/language.js

```diff
@@ -0,0 +1,54 @@
+import store from '../store';
+
+/**
+ * To normalize Language String and return language code
+ * @param {String} languageString
+ */
+const normalizeLanguageString = (languageString) => {
+	let [languageCode, countryCode] = languageString.split ? languageString.split(/[-_]/) : [];
+	if (!languageCode || languageCode.length !== 2) {
+		return 'en';
+	}
+	languageCode = languageCode.toLowerCase();
+
+	if (!countryCode || countryCode.length !== 2) {
+		countryCode = null;
+	} else {
+		countryCode = countryCode.toUpperCase();
+	}
+
+	return countryCode ? `${ languageCode }_${ countryCode }` : languageCode;
+};
+
+/**
+ * To get browser Language of user
+ */
+const browserLanguage = () => {
```

Comment:
```suggestion
const browserLanguage = () => (navigator.userLanguage || navigator.language);
```

Use the correct arrow function statement. Your function does not have a return.